tests/calc/test_cross_sections.py

```diff
@@ -148,7 +148,7 @@ def test_distances_from_cross_section_given_xy(test_cross_xy):
 def test_distances_from_cross_section_given_bad_coords(test_cross_xy):
     """Ensure an AttributeError is raised when the cross section lacks neeed coordinates."""
```

Comment:
Not your change, but looks like we dropped a letter there.

Reply:
By all means put in a PR for that.

Reply:
Hey, it's cross sections, so the `git blame` is probably mine anyways!